example/index.html

```diff
@@ -1,20 +1,24 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <script src="https://sdk.yoti.com/clients/browser.2.0.1.js"></script>
         <meta charset="utf-8">
         <title>Your Site</title>
+
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+
+        <script src="https://sdk.yoti.com/clients/browser.2.0.1.js"></script>
     </head>
     <body>
         <h1>Your Site Information</h1>
         <span data-yoti-application-id="your App ID"
             data-yoti-type="inline"
-            data-yoti-scenario-id="your Scenario ID"
-            data-size="small">
-         Use Yoti!
+            data-yoti-scenario-id="your Scenario ID">
+         USE YOTI!
```

Comment:
Please don't use unnecessary uppercase and exclamation marks `Use Yoti` on the button will do.

Reply:
I used uppercase to be inline with the QR code button as it will display the text in uppercase anyway.